api/tests/unittest/test_feeds.py

```diff
@@ -125,20 +125,7 @@ def test_gtfs_feeds_get(client: TestClient, mocker):
         subdivision_name="test_subdivision_name",
         municipality="test_municipality",
     )
-    mock_bounding_box = json.dumps(
```

Comment:
[suggestion]: to have better coverage, I suggest creating a test for the missing bounding box, leaving this test as it covers the bounding box's presence and structure.

Reply:
Actually AFAIK the bounding box is not checked in this test. 
I could add an assert for the bounding box in this test and create a new one.

Reply:
Yes please